src/content/docs/journey-migration/making-the-switch.mdx

```diff
@@ -0,0 +1,127 @@
+        * Install browser agents as quickly as you would any other using our guided installations.
+        * Find out what's causing issues with how customers interact with your site from the beginning to the end of a transaction.
+        * Monitor your AJAX requests, JavaScript errors, and more as they occur in real time so you can resolve any issues before they become problems.
+        * Customize your monitoring charts and share them across your organization so everyone has the same context you do.
+    Learn more about browser monitoring in our [short overview](https://newrelic.com/platform/browser-monitoring), or see our [introduction to browser monitoring](/docs/browser/browser-monitoring/getting-started/introduction-browser-monitoring/) documentation.
+    </Collapser>
+    <Collapser
+        id="d-tracing"
+        title="Distributed tracing"
+    >
+    Get every data point from every single trace in one centralized location. Find only what you need among your trace when you need it, and eliminate hunting through pages of data to identify issues when they arise. 
+        * Automatically instrument distributed tracing to start observing your data almost instantly.
+        * Utilize the seemless integration with our APM agents and let the two features work in tandem to reduce your workload.
+        * Capture representative activity samples with our head-based sampling approach.
+        * Take advantage of our **Infinite Tracing**, a feature that users tail-based sampling approaches that gathers information at the end of a trace and lets you know what's most important.
+    Learn more about distributed tracing in our [introduction to distributed tracing](/docs/distributed-tracing/concepts/introduction-distributed-tracing/) documentation.       
+    </Collapser>
+    <Collapser
+        id="NRQL"
+        title="NRQL: New Relic's query language"
+    >
+    Maximizing your data requires the ability to query what you need down to the smallest detail, and that's exactly what New Relic's query language (NRQL) provides. Similar to ANSI SQL, you can use our language as a foundation to get any number of details about your data in our platform.
+        * Build charts, dashboards, and alerts from anywhere in our platform using the NRQL console. 
+        * Take advantage of our free tutorials and New Relic University courses to learn everything you need to know about the language. 
+        * Explore your data in ways no other platform can provide and get every detail on any aspect of observability.
+        * Use NRQL to make queries and perform specific configurations via NerdGraph, our GraphQL-format API.
+    Learn more about [how NRQL works](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-how-nrql-works/), or see our [introduction to NRQL](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language/) documentation.       
```

Comment:
Same as browser, gets combined with previosu bullet.

Reply:
Should be fixed